core/src/main/scala/org/apache/gearpump/cluster/master/MasterSummary.scala

```diff
@@ -23,7 +23,14 @@ object MasterStatus {
   val UnSynced = "unsynced"
 }
 
-case class MasterDescription(leader: (String, Int), cluster: List[(String, Int)], aliveFor: Long,
-                             logFile: String, jarStore: String,
-                             masterStatus: MasterStatus.Type,
-                             homeDirectory: String)
+case class MasterSummary(
+  leader: (String, Int),
+  cluster: List[(String, Int)],
+  aliveFor: Long,
+  logFile: String,
+  jarStore: String,
+  masterStatus: MasterStatus.Type,
+  homeDirectory: String,
+  activities: List[MasterActivity])
+
+case class MasterActivity(time: Long, event: String)
```

Comment:
I don't know what is the best approach. 

1. we don't want to create a package named shared, that means we need to copy everything under this package.
2. We don't want to modify the copy-logic for every new class file we created.